### ISSUE TITLE
Custom cookie to http-request

### ISSUE THREAD

**Issue by TuomasK1989** [CesiumGS/cesium-unity#146]
Hi, 
I would like to stream 3D Tiles from AWS with the "Tileset Source" -Option set to "From Url". However my content is private and I use custom cookies in http request header to authenticate request. Is it possible to set custom cookies or somehow customize http requests in Unity/c# side or how can I achieve this?

Br, 
TMK

**Comment by kring** [comment]
@TuomasK1989 that's not currently possible without modifying Cesium for Unity itself, but we'll consider it an enhancement request.

The best you can do right now is include authentication information in query parameters.

**Comment by TuomasK1989** [comment]
Hi,
I noticed that there has been some development considering request headers. Is it possible to use custom headers in the current or upcoming release?

**Comment by kring** [comment]
No, the recent work made sure headers added to previous requests could be retrieved from the request object. Adding the ability to specify request headers via configuration on the tileset isn't on our near-term roadmap, but we'd welcome a community contribution to add it. It should be relatively straightforward.
